app/models/api/v2/form.rb

```diff
@@ -0,0 +1,30 @@
+# Currently this is just a facade around the v1 Form model,
+# as it uses the same database table
+class Api::V2::Form < ApplicationRecord
+  include Rails.application.routes.url_helpers # TODO: do we need a presenter instead?
```

Comment:
I like creating a new Form object using the same table! 

I think a present is something to think about. Including urls in models seems like a rails code smell of some sort but it's pragmatic and simple.

Reply:
> I think a present is something to think about. Including urls in models seems like a rails code smell of some sort but it's pragmatic and simple.

I agree it's a code smell, I did look at the [Draper decorator gem](https://github.com/drapergem/draper), and there are ways to make presenters without an extra dependency (see https://www.rubyguides.com/2019/09/rails-patterns-presenter-service/ for instance), but because it doesn't seem to be a built-in Rails thing I didn't want to go too far with the idea in this PR.